### ISSUE TITLE
Create a e2e solution for testing the PDF creation

### ISSUE THREAD

**Issue by mjknight50** [department-of-veterans-affairs/va.gov-team#72423]
# Problem
The classic issue is that the front-end and back-end don't share a common data validation fixture.

We want to be able to verify that our e2e tests can completely fill out a form and generate a PDF as part of the CI/CD process.  Currently there are three ideas for a potential common fixture.  We need to evaluate which makes the most sense to use going forward.

1.  [Maximal Test](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/pensions/tests/schema/maximal-test.json
) - The maximal test file is used for cypress to validate the form.  

2.  [Form Filler](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/1c22596850072bc57a55ce707ec62ecb94290a6e/teams/benefits/scripts/pensions/21P-527EZ-Form-Filler.user.js) - The form filler was designed to facilitate front-end testing, but not originally intended for PDF validation.

3.  [JSON Schema](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/21P-527EZ/schema.js) - The JSON schema was likely intended to be the link between the front and back end.

# Goal
Decide which is the best method to link FE and BE.  Create a test process to verify the front-end can be filled out completely and then a PDF can be generated from it.

**Comment by vbellissimo** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/department-of-veterans-affairs/va.gov-team/72423?planning-poker) with Zenhub @danlim715 @scottsdevelopment @TaiWilkin @ToddWebDev @wayne-weibel @mjknight50

**Comment by TaiWilkin** [comment]
@mjknight50 Some thoughts on this:
in `vets-json-schema`, there is a folder called 'examples'. These examples are pulled into some tests in `vets-api` (as `VetsJsonSchema::EXAMPLES`). I don't see any used in `vets-website` tests, but I was able to access the examples there when I checked. I wonder if it would work to move our maximal, simple, and overflow tests to be examples in `vets-json-schema` and then use those in our frontend and backend tests.
